roles/elastic-stack/ansible-elasticsearch/tasks/Debian.yml

```diff
@@ -1,12 +1,10 @@
 ---
 - name: Debian/Ubuntu | Install apt-transport-https and ca-certificates
   apt:
-    name: "{{ item }}"
+    pkg:
```

Comment:
I would recommend using `name`, since `pkg` and `package` are aliases. I suspect they might be getting phased out at some point. Also, name is more commonly used.